src/wallet/rpcwallet.cpp

```diff
@@ -4037,6 +4037,7 @@ extern UniValue importprivkey(const UniValue& params, bool fHelp);
 extern UniValue importaddress(const UniValue& params, bool fHelp);
 extern UniValue importpubkey(const UniValue& params, bool fHelp);
 extern UniValue dumpwallet_zcoin(const UniValue& params, bool fHelp);
+extern UniValue dumpmnemonic(const UniValue& params, bool fHelp);
```

Comment:
I also think an `importmnemonic` function is needed. Make sure to check for wallet version

Reply:
I think better to allow  inputting mnemonic only during first startup.